docs/proposal-rose-suite-run.md

```diff
@@ -84,11 +84,58 @@ configuration logic. Some points to consider:
     * Backward (message and manipulation commands).
 * A special section for configuring the suite cluster?
 * Deprecate `suite.rc` relevant settings under `[runtime][__MANY__]`, but have
-  a top level `[clusters]` section that simply mirror that of `global.rc` to
+  a top level `[job platforms]` section that simply mirrors that of `global.rc` to
   allow individual suites to override the site/user settings.
 * Users will configure tasks to run on clusters instead of hosts/batch systems.
 * If relevant, improve alignment with DRMAA Open Grid Forum API?
 
+So, for example, a `suite.rc` might look like this:
+```ini
+[cylc]
+    ...
+
+[scheduling]
+    ...
+
+[job platforms]
+    [[spice]]
+        [[[batch system]]]
+            class = slurm
+            before command = sshare -u %(user_id)
+            after command = sacct -j %(jobid)s --format=""
+    [[xcs]]
+        login hosts = xcslr0, xcslr1
+        retrieve job logs = True
+        [[[batch system]]]
+            class = pbs
+        [[[default directives]]]
+            --some-directive
+    [[desktop]]
+        login hosts = vld392
+
+[runtime]
+    [[Alice]]
+        ...
+
+    [[Bob]]
+        ...
+        [[[platform]]]
+            platform = desktop
```

Comment:
I think we'll merge the old `[[[remote]]]` into the `[[[job]]]` section. So:

```
[[[job]]]
    platform = ...
```

Reply:
Sounds very sensible